### ISSUE TITLE
Label "org.label-schema.vcs-url" is missing

### ISSUE THREAD

**Issue by hugocf** [rubendg/sbt-docker-label-schema#2]
The labels generated for the Docker images are all fine, except for the `vcs-url` which is missing. 

```shell
$ docker inspect form-deployer:0.7.78-SNAPSHOT | jq '.[].Config.Labels'
{
  "org.label-schema.build-date": "2018-02-26T18:21:24.406Z",
  "org.label-schema.description": "form-deployer",
  "org.label-schema.name": "form-deployer",
  "org.label-schema.schema-version": "1.0",
  "org.label-schema.vcs-ref": "b1a22fba8f01831c3c91ca2c73206ab2dc108f09",
  "org.label-schema.vendor": "com.uxforms",
  "org.label-schema.version": "0.7.78-SNAPSHOT"
}
```

Looking at line [DockerLabelSchemaPlugin.scala#L55](https://github.com/rubendg/sbt-docker-label-schema/blob/master/src/main/scala/nl/rubendegooijer/sbt/DockerLabelSchemaPlugin.scala#L55):

```scala
      vcsUrl = Seq(scmInfo.value.map(_.browseUrl), Git.native.remoteUrl).pickFirst,
```

… and considering I don’t have any `scmInfo` key defined, it seems that `Git.native.remoteUrl` is failing to retrieve the url and returning `None`.

When the command it runs is executed directly in the terminal, it returns a String in the expected format:

```shell
$ git config --get remote.origin.url
git@bitbucket.org:uxforms/uxforms-form-deployer.git
```

So something must be preventing the `remoteUrl` functions to retrieve the url or the `remoteUrl` Process call is failing for some reason…

Any idea of what might be going on or how can I help debug this further?

---

* `scalaVersion := "2.11.9"`
* `sbt.version = 0.13.15`
* `"sbt-native-packager" % "1.3.3"`

**Comment by rubendg** [comment]
Good catch. I now consistently applied the removal of trailing newlines from the output (https://github.com/rubendg/sbt-docker-label-schema/commit/ef154ef027c61a7f411f0736c513259f2002d031). My best guess is that that is causing the problem. Could you give it a test by cloning the repo and running `sbt publishLocal`, if that solves it I'll release v0.1.1.

**Comment by hugocf** [comment]
Thanks a lot, that’s great! Yes, I can confirm it works now.

I couldn’t get the `sbt publishLocal` version to work with my existing repos because the locally published version was only for Scala `2.12`, but starting a new Play app from scratch a could reproduce the problem & the solution correctly.

```sbt
$ cat project/docker-label-schema.sbt 
resolvers += Resolver.bintrayIvyRepo("rubendg", "sbt-plugins")

addSbtPlugin("nl.rubendegooijer.sbt" % "sbt-docker-label-schema" % "0.1.0-3-gef154ef")
```

```json
$ docker inspect 35d14198e171 | jq '.[].Config.Labels'
{
  "org.label-schema.build-date": "2018-02-27T19:51:28.764Z",
  "org.label-schema.description": "play-app",
  "org.label-schema.name": "play-app",
  "org.label-schema.schema-version": "1.0",
  "org.label-schema.vcs-ref": "57122a3ca06ca9141b4533063772595d56d918d3",
  "org.label-schema.vcs-url": "https://github.com/hugocf/play-app.git",
  "org.label-schema.vendor": "com.example",
  "org.label-schema.version": "1.0-SNAPSHOT"
}
```

**Comment by rubendg** [comment]
I see. `sbt ^publishLocal` will cross publish as scala 2.10 and 2.12, that should work for you. I've released v0.1.1 that includes the fix.

**Comment by hugocf** [comment]
> `sbt ^publishLocal` will cross publish as scala 2.10 and 2.12 (…)

Ah, nice! Didn't know that one. Thanks
